config/general.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/utils"
+	"os"
+	"path/filepath"
+)
+
+const (
+	JfrogCurationDirName = "curation"
+
+	CurationsDir         = "JFROG_CLI_CURATION_DIR"
+	CurationMavenSupport = "JFROG_CLI_CURATION_MAVEN"
```

Comment:
```suggestion
         // #nosec G101 -- Not credentials.
	CurationMavenSupport = "JFROG_CLI_CURATION_MAVEN"
```
To fix the GO-Sec add the following, change the reason if needed.